packages/web-extension/src/messaging/runContentScriptMessageProxy.ts

```diff
@@ -23,9 +24,16 @@ export const runContentScriptMessageProxy = (apis: AnyApi[], logger: Logger) =>
     const apiFunction = apis.find((api: any) => typeof api[data.request.method] === 'function')?.[data.request.method];
     if (!apiFunction) return;
 
+    let response;
+    try {
+      response = await apiFunction(...data.request.args);
+    } catch (error) {
+      response = toSerializableObject(error);
+    }
+
     const responseMessage: ResponseMessage = {
       messageId: data.messageId,
-      response: await apiFunction(...data.request.args)
+      response
```

Comment:
cip30 responses might not have any objects that need this right now, but it might save us from a sneaky bug in the future
```suggestion
      response: toSerializableObject(response)
```

Reply:
fixed here: [34fa24d](https://github.com/input-output-hk/cardano-js-sdk/pull/704/commits/34fa24d0a3cdae21f89b41f4b85d9de34f2e1220)